wallet/lib/pages/purchase_item/widgets/pay_now_dialog.dart

```diff
@@ -129,78 +118,56 @@ class _PayNowWidgetState extends State<PayNowWidget> {
-                      )),
-                      height: 1.sh,
-                      fit: BoxFit.fitHeight,
-                      placeholder: (context, _) => Shimmer(
-                        color: PylonsAppTheme.cardBackground,
-                        child: Container(),
-                      ),
-                    ),
-                  ),
-                  title: Text(
-                    widget.nft.name,
-                    style: _titleTextStyle,
-                    maxLines: 1,
-                  ),
-                  subtitle: Column(
-                    crossAxisAlignment: CrossAxisAlignment.stretch,
-                    mainAxisSize: MainAxisSize.min,
-                    children: [
-                      Text(
-                        "${"created_by".tr()} ${widget.nft.creator}",
-                        style: _subtitleTextStyle,
-                      ),
-                      if (widget.nft.owner.isNotEmpty)
-                        Text(
-                          "${"sold_by".tr()} ${widget.nft.owner}",
-                          style: _subtitleTextStyle,
-                        ),
-                    ],
-                  ),
+                SizedBox(
+                  height: 60.h,
                 ),
+                Padding(
+                    padding: EdgeInsets.only(left: 20.w),
+                    child: Text("checkout".tr(), style: _titleTextStyle)),
                 SizedBox(
                   height: 30.h,
                 ),
                 buildRow(
-                  subtitle: widget.nft.ibcCoins.getAbbrev(),
-                  title: "currency".tr(),
+                  subtitle: widget.nft.name,
+                  title: "NFT",
                 ),
                 SizedBox(
                   height: 3.h,
                 ),
                 buildRow(
-                  subtitle: widget.nft.ibcCoins.getCoinWithDenominationAndSymbol(price.toString(), showDecimal: true),
-                  title: "price".tr(),
+                  subtitle: "#${widget.nft.amountMinted} of ${widget.nft.quantity}",
+                  title: "edition".tr(),
                 ),
                 SizedBox(
                   height: 3.h,
                 ),
-                buildPylonsFeeRow(
-                  subtitle: widget.nft.ibcCoins.getCoinWithDenominationAndSymbol(fee.toString(), showDecimal: true),
-                  title: "pylons_fee".tr(),
+                buildRow(
+                  subtitle: widget.nft.creator,
+                  subtitleTextColor: AppColors.kTradeReceiptTextColor,
+                  title: "artist".tr(),
                 ),
                 SizedBox(
                   height: 3.h,
                 ),
-                buildRow(
-                  subtitle: widget.nft.ibcCoins.getCoinWithDenominationAndSymbol(widget.nft.price, showDecimal: true),
-                  title: "total".tr(),
-                ),
+                if(widget.nft.owner.isNotEmpty)
```

Comment:
Why owner will be empty?

Reply:
> @samarali92 if you look closely both designs are same. I took the screenshot from the figma design .

It seems like I was following another figma design for the pay now dialog.

Reply:
It was already like this before in the code. https://github.com/Pylons-tech/pylons/blob/main/wallet/lib/pages/purchase_item/widgets/pay_now_dialog.dart Line no. 167

Reply:
remove it.